### DISCUSSION TITLE
Dock.dev Environment Collection Editor Question Set Update Not Working

### DISCUSSION THREAD

**Discussion by ramkumar-ns-dev** [Sunbird-Knowlg/Community#170] [Bugs]
### **Prerequisites**
I have searched for existing issues that already report this problem, without success.

### **Current Behaviour**
I'm trying to save Question set as Draft of Observation with rubrics in dock dev but it's throwing 500 internal server error.

### **Expected Behaviour**
When user try to save Question set as Draft it should need to save Question set without failure.

### **Steps to Reproduce**
	1. Created project from Sourcing admin.
	2. Submitted nomination from contributor.
	3. Approve Nomination from sourcing admin.
	4. Click "Create/upload content" on project page it will open Question set page with domain and criteria.
	5. Add Questions to the Question set.
	6. Then we click on "save as Draft" button instead of saving the Question set it's failing from following API.
	7. Failing API => https://dock.sunbirded.org/api/question/v1/list
	
### **API Endpoints used**
       1. https://dock.sunbirded.org/api/question/v1/list
        
### Additional Information
Tested on Google Chrome: Version 114.0.5735.198 (Official Build) (arm64)

**Note** : I'm running the dock dev Environment in Local setup.
can anybody help on this.
@vaivk369 

cc: @kiranharidas187

**Comment by pallakartheekreddy** [comment]
@ramkumar-pacewisdom Can you please provide more details to help, and please raise this request in below repo as this issue with coKreat.
https://github.com/orgs/Sunbird-coKreat/discussions

**Reply by ramkumar-ns-dev** [reply]
@pallakartheekreddy sure, will raise this in https://github.com/orgs/Sunbird-coKreat/discussions following repo.
